### ISSUE TITLE
minescript.getInventory().getSlot(0).damageValue Always == -1

### ISSUE THREAD

**Issue by Avidhero** [michaelzangl/minebot#59]
Suppose it stems from /Minebot/src/net/famzangl/minecraft/minebot/ai/strategy/InventoryDefinition.java
Line 85-87.
```javascript
slots.add(new InventorySlot(i, stack.stackSize, Item
					.getIdFromItem(stack.getItem()),
					stack.getHasSubtypes() ? stack.getItemDamage() : -1));
```

Anyways, when trying to write a script to access the durability of a pickaxe when mining, minescript.getInventory().getSlot(0).damageValue  seems to always give me a -1, regardless of the damage on the item.

**Comment by michaelzangl** [comment]
Yes. It was not really meant to be the damageValue but the subtype.

I'll change this to the ItemWithSubtype class soon. Then I'll also add methods to get the item type, material and enchantments (and let you query an item rater formula on them, e.g. "iron&pickaxe")

**Comment by Avidhero** [comment]
Thanks! It should enable much more valuable scripts! Loving this bot, so powerful!
